airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -687,13 +706,13 @@
         currentStreamUrl = song.streamUrl;
         updateCurrentImage();
 
-        // Handle ChromeCast player.
-        if (CastPlayer.castSession) {
+<c:if test="${model.googleCast}">
```

Comment:
The mixing of JSP and JS conditions is hard to read. How about doing something like setting the Google Cast instance to `null` at initialization if it's not enabled, and checking on that in JS only?